supportedBackends/jes/src/test/scala/cromwell/backend/impl/jes/JesJobExecutionActorSpec.scala

```diff
@@ -83,7 +84,14 @@ class JesJobExecutionActorSpec extends TestKitSuite("JesJobExecutionActorSpec")
     // Wait for the JABJEA to be spawned. Then kill it:
     parent.expectNoMsg(max = AwaitAlmostNothing)
     deathwatch.expectNoMsg(max = AwaitAlmostNothing)
-    jabjeaConstructionPromise.future foreach { _ ! JabjeaExplode }
+    jabjeaConstructionPromise.future onComplete {
+      case Success(jabjea) =>
+        jabjea ! JabjeaExplode
+      case Failure(throwable) =>
+        val exception = new RuntimeException("Error creating jabjea for test!", throwable)
+        exception.printStackTrace()
```

Comment:
Yup. Should never error, but if/when it does, it won't just silently fail. I was eliminating all uses of `Future.foreach`.